flask_jwt_extended/view_decorators.py

```diff
@@ -32,12 +32,13 @@ def jwt_required(fn):
     """
     @wraps(fn)
     def wrapper(*args, **kwargs):
-        jwt_data = _decode_jwt_from_request(request_type='access')
-        ctx_stack.top.jwt = jwt_data
-        if not verify_token_claims(jwt_data[config.user_claims_key]):
-            raise UserClaimsVerificationError('User claims verification failed')
-        _load_user(jwt_data[config.identity_claim_key])
-        return fn(*args, **kwargs)
+        if request.method not in config.exempt_methods:
```

Comment:
Could we add this same logic to the other view decorators in this file as well, for completeness sake? I'm alright with having the unit tests be only for the `@jwt_required` decorator (unless you feel like adding unit tests for the other decorators as well, in which case 👍 👍)

Reply:
Yea! I'll add it also in `fresh_jwt_required`! I'm not sure if we need to implement it in `jwt_optional`, since it should works well!

I'll wrote more tests anyway :D